src/kv/Makefile.am

```diff
@@ -26,14 +26,12 @@ NPROC = nproc
 endif
 
 # explicitly disable jemalloc unless we are using it.
-if WITH_JEMALLOC
-DISABLE_JEMALLOC = 0
-else
-DISABLE_JEMALLOC = 1
+if !WITH_JEMALLOC
+export DISABLE_JEMALLOC=1
 endif
 
 rocksdb/librocksdb.a:
-	cd rocksdb && CC="${CC}" CXX="${CXX}" EXTRA_CXXFLAGS="${ROCKSDBCXX_FLAGS}" PORTABLE=1 DISABLE_JEMALLOC="${DISABLE_JEMALLOC}" ${MAKE} -j$(shell ${NPROC}) static_lib
+	cd rocksdb && CC="${CC}" CXX="${CXX}" EXTRA_CXXFLAGS="${ROCKSDBCXX_FLAGS}" PORTABLE=1 ${MAKE} -j$(shell ${NPROC}) static_lib
```

Comment:
@cbodley 

automake actually **always** disables JeMalloc even it is enabled using "--with-jemalloc". but we don't pass this in our gitbuilders.